src/library/eventStream.js

```diff
@@ -17,6 +17,8 @@ import { parseJson } from './util';
  * @property {MessageAttribute<'trigger' | 'pass' | 'fail'>} status
  */
 
+// CR: Mickey: Motivation to have isString thru getAttrType here and not in util.js?
+
```

Comment:
@MickeyMcc Absolutely no reason or motivation. I will move this as well as any such common checks to util.js